applications/nrf5340_audio/Kconfig.sysbuild

```diff
@@ -4,7 +4,37 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
-
 config NRF_DEFAULT_IPC_RADIO
 	default y
+
+config AUDIO_DFU_EXTERNAL_FLASH
+	bool "Audio Application DFU using external flash"
+	default n
```

Comment:
This should be default y? And, there must be a big fat warning if setting this to n.

Reply:
This doesn't select between internal or external, it simply enables external. So if this is set to yes it will include MCUboot and B0N, and require an external flash shield, which is not the intention.